### PR TITLE
test: add integration tests for paytm modals

### LINKED ISSUE #3468: Add integration tests for new modals
This is a follow up for the addition of integration tests for new modal components to increase the test coverage.

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #3468 

#### Short description of what this resolves:
Add integration tests for paytm modals\

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by uds5501** [comment]
@kushthedude @mrsaicharan1 Please review

**Comment by uds5501** [comment]
This needs to be updated once #3476 is updated

**Comment by uds5501** [comment]
@Anupam-dagar I will update this PR today itself and ping you to review.

**Comment by uds5501** [comment]
@shreyanshdwivedi @mrsaicharan1 @kushthedude @Anupam-dagar Please review

File: tests/integration/components/modals/paytm-otp-test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+module('Integration | Component | modals/paytm otp modal', function(hooks) {
+  setupIntegrationTest(hooks);
+
+  test('it renders', async function(assert) {
+    this.setProperties({
+      'isOpen'    : false,
+      'currency'  : 'USD',
+      'amount'    : 100,
+      'verifyOtp' : () => {}
```
Comment (mrsaicharan1) [inline tests/integration/components/modals/paytm-otp-test.js]: This looks fine but what does this test exactly?

Comment (uds5501) [reply tests/integration/components/modals/paytm-otp-test.js]: In the current state, it checks rendering of the modal without actually hitting the API (gotta save the API calls xD  )

**Pull request merged**